easybuild/easyblocks/b/bzip2.py

```diff
@@ -40,14 +42,32 @@ class EB_bzip2(ConfigureMake):
     def configure_step(self):
         """Set extra configure options (CC, CFLAGS)."""
 
+        self.cfg.update('prebuildopts', "make -f Makefile-libbz2_so && ")
         self.cfg.update('buildopts', 'CC="%s"' % os.getenv('CC'))
         self.cfg.update('buildopts', "CFLAGS='-Wall -Winline %s -g $(BIGFILES)'" % os.getenv('CFLAGS'))
 
     def install_step(self):
         """Install in non-standard path by passing PREFIX variable to make install."""
 
         self.cfg.update('installopts', "PREFIX=%s" % self.installdir)
+        
+        srcdir = self.cfg['start_dir']
+        destdir = os.path.join(self.installdir, 'lib/')
+        os.mkdir(destdir)
+        dynamic_libs_to_copy = glob.glob('libbz2.so.*')
 
+        # copy dynamic libraries to install_dir/lib
+        try:
+            for lib in dynamic_libs_to_copy:
+                os.system('mv %s %s' % (lib, destdir)) # no easy way to copy sysmlinks using python shutil?
```

Comment:
bleh, yeah, seems like you're right... http://stackoverflow.com/questions/4847615/copying-a-symbolic-link-in-python